packages/lavapack/test/runtimeEcma.spec.js

```diff
@@ -0,0 +1,14 @@
+const test = require('ava')
+const fs = require('node:fs')
+const espree = require('espree')
+
+const ECMA_VERSION = 2020;
+
+test('runtime.js parses as ECMA 2020', (t) => {
+  const code = fs.readFileSync(require.resolve('../src/runtime.js'), 'utf8')
+  t.notThrows(() => {
+    espree.parse(code, {
+    ecmaVersion: ECMA_VERSION,
+    })
```

Comment:
(should) This test used to be more informative. 
It's likely that if this test fails, it does so unexpectedly and a more descriptive message (ie containing the runtime.js` filename) is going to be helpful for the person encountering it.

Reply:
I'll restore the custom failure output, but keep it in an ava test.